shared/actions/json/chat2.json

```diff
@@ -67,7 +67,7 @@
         "'inboxSyncedUnknown'",
         "'joinedAConversation'",
         "'leftAConversation'",
-        "'teamTypeChanged'",
```

Comment:
i guess your linter doens't like dangling commas?

Reply:
This file is just auto generated right?

Reply:
opposite, this is the source of the generation. its fine though, just mentioning it

Reply:
Ah right, I guess it did make a bunch of changes.